src/Symfony/Component/Yaml/Parser.php

```diff
@@ -359,6 +308,47 @@ public function parse($value, $flags = 0)
             mb_internal_encoding($mbEncoding);
         }
 
+        if (array_key_exists('<<', $data)) {
+            $merged = $data['<<'];
```

Comment:
In case of multiple merge keys, this would be an array of arrays and we will need to traverse it. Always ensuring that `$data['<<']` is an array would make the traversing easier and would allow to use `isset()`.